e2e_playwright/test_assets/hostframe.html

```diff
@@ -41,6 +41,10 @@
 <script>
   window.addEventListener("message", (event) => {
     console.log("Received message event:", event);
+
+    if (event.data.type === "REDIRECT_TO_URL") {
+      window.location.href = event.data.url
+    }
```

Comment:
This was only added to the deprecated hostframe.html, but not 100% sure if really required.